src/SDK/MonitoringApplianceEmulator/Models/SmartDetectorRunner.cs

```diff
@@ -371,5 +399,23 @@ private List<ResourceIdentifier> GetTargetResourcesForDetector(HierarchicalResou
 
             return workspaceIdToWorkspaceResourceIdMapping;
         }
+
+        /// <summary>
+        /// Gets a valid log name for running the detector on the specified emulated time
+        /// </summary>
+        /// <param name="emulatedTime">The detector's run emulated time.</param>
+        /// <returns>The log name for the run.</returns>
+        private string GetValidLogName(DateTime emulatedTime)
+        {
+            string logName = $"{emulatedTime:yyyy-MM-dd HH-mm}";
```

Comment:
>logName [](start = 19, length = 7)

why not also including the detector name for a better archive?

Reply:
or maybe to include it in the zip archive file name.. otherwise, do we want to be able to load logs of detector X when the emulator is working for detector Y?

---
In reply to: [231554507](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/13#discussion_r231554507) [](ancestors = 231554507)